### ISSUE TITLE
CLICKING NEW MODULE DOES NOTHING WINDOWS ANDROID STUDIO

### ISSUE THREAD

**Issue by gptshubham595** [multi-os-engine-community/moe-ide-integration#2]
NO IOS MODULE IS CREATED 

![image](https://user-images.githubusercontent.com/24877361/94156765-14794080-fe9e-11ea-914f-1d2cc5df4318.png)

**Comment by Berstanio** [comment]
If this is still an issue, I think it is a problem with the Android Studio Kotlin-Plugin. If you downgrade the plugin, didn't tested other versions, to 1.3.72(https://plugins.jetbrains.com/plugin/6954-kotlin/versions), the dialog should work again.

**Comment by Noisyfox** [comment]
Yes it's a bug in Kotlin plugin: https://youtrack.jetbrains.com/issue/KT-42779
For now just never upgrade the kotlin plugin version, always use the bundled version came with Android Studio.
